docs/source/model_design/designing_vivarium_model/fertility/index.rst

```diff
@@ -0,0 +1,151 @@
+.. _fertility_crude_birth_rate:
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  ---------------------
+
+  Section Level 2 (#.#)
+  +++++++++++++++++++++
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+===================================================
+Fertility Component that uses Crude Birth Rate Data
```

Comment:
It feels weird to me to say this "uses Crude Birth Rate Data," when, as described in "GBD modeling strategy," the data we get for this is _not_ a CBR, it is a count of "live births by sex."

Perhaps this component, both here and in [`vivarium_public_health`](https://github.com/ihmeuw/vivarium_public_health/blob/fcb9f4e3ec318a5b6d8f12fdde4308b5af9c0878/src/vivarium_public_health/population/add_new_birth_cohorts.py#L76), is misnamed?

Would be interested to hear others' thoughts.